src/utils/extractVideoEmbedFromStory.ts

```diff
@@ -0,0 +1,16 @@
+const extractVideoEmbedFromStory = (content) => {
```

Comment:
Need a return type on this. https://www.typescriptlang.org/docs/handbook/functions.html#typing-the-function
```suggestion
const extractVideoEmbedFromStory = (content): string => {
```